### PR TITLE
Permit mm-grid without recycling

### PR DESCRIPTION
**Pull request by jcmoore**
@dlasky 
@anthonykoerber

### CODE REVIEW

File: src/mm-item-recycler/mm-item-recycler.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 -->
 <link rel="import" href="../../bower_components/polymer/polymer.html">
-<polymer-element name="mm-item-recycler" on-scroll="{{scrollHandler}}">
+<polymer-element name="mm-item-recycler" on-scroll="{{scrollHandler}}" recycle="true">
```
Comment (dlasky) [inline src/mm-item-recycler/mm-item-recycler.html]: shouldn't this be in attributes list?

**Comment by dlasky** [comment]
+1

**Comment by anthonykoerber** [comment]
@jcmoore - let's chat when you're in. While this works, the grid doesn't actually get a calculated height, because of the overflow:visible... so while it will display, because of absolute positioning, it won't have the correct metrics within the page flow (ie - other items will stack over/under it).

Tested this with 16 items - There are a total of 4 recycler panels, the 4th has 4 additional "empty" items to get the total up to 5(?) Not sure if this is by design or not.

File: src/mm-item-recycler/mm-item-recycler.html (hunk 1 above)
Comment (jcmoore) [inline src/mm-item-recycler/mm-item-recycler.html]: whoops, been awhile

**Comment by dlasky** [comment]
+1 on the updates as well (thanks for exposing resize)

**Comment by anthonykoerber** [comment]
+1 looks greeeeeat! Pushes layout below as expected. Ship.

**Comment by dlasky** [comment]
+1 on the latest

**Pull request merged**